torch/csrc/jit/passes/onnx/constant_map.cpp

```diff
@@ -0,0 +1,110 @@
+#include <torch/csrc/jit/passes/onnx/constant_map.h>
+
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+namespace torch {
+namespace jit {
+
+namespace onnx {
+using namespace ::c10::onnx;
+}
+
+// Meyer’s Singleton for C++ 14
+ConstantValueMap& ConstantValueMap::getInstance() {
+  static ConstantValueMap s;
+  return s;
+}
+
+void ConstantValueMap::SetRank(
+    const std::string& tensorName,
+    size_t rankValue) {
+  ConstantValueMap::getInstance().rankMap.emplace(tensorName, rankValue);
+}
+
+bool ConstantValueMap::HasRank(const std::string& tensorName) {
```

Comment:
nit: add const to this function

Reply:
This is static function, so I could not add const to it.